addons/mail/static/src/discuss/core/common/message_patch.js

```diff
@@ -0,0 +1,24 @@
+/* @odoo-module */
+
+import { Message } from "@mail/core/common/message";
+import { patch } from "@web/core/utils/patch";
+
+patch(Message.prototype, {
+    /** @override */
+    enterEditMode() {
+        const body = document.createElement("span");
+        body.innerHTML = this.props.message.body;
```

Comment:
This could use the DOMParser instead of DOM manipulations:
```suggestion
        const body = new DOMParser().parseFromString(this.props.message.body, "text/html");
```